contrib/idxd-config-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+## accel_config is the utility library required by QPL-Deflate codec for controlling and configuring Intel® In-Memory Analytics Accelerator (Intel® IAA).
+option(ENABLE_ACCEL_CONFIG "Enable accel-config library" ${ENABLE_LIBRARIES})
+
+if (NOT ENABLE_QPL)
```

Comment:
Re my other comment about switching

```
add_contrib (qpl-cmake qpl)
add_contrib (idxd-config-cmake idxd-config)
```

... when doing so, access to ENABLE_QPL (l. 4) will no longer work then from here. Also, the accel-config build should rather not depend on `ENABLE_LIBRARIES` (l. 2), we should avoid a situation where accel-config is build but QPL isn't.

So maybe do this:
```cmake
# in contrib/CMakeLists:

[...]
if (OS_LINUX AND ARCH_AMD64 AND (ENABLE_AVX2 OR ENABLE_AVX512))
    option (ENABLE_QPL "Enable Intel® Query Processing Library" ${ENABLE_LIBRARIES})
elseif(ENABLE_QPL)
    message (${RECONFIGURE_MESSAGE_LEVEL} "QPL library is only supported on x86_64 arch with avx2/avx512 support")
endif()
if (ENABLE_QPL)
    add_contrib (idxd-config-cmake idxd-config)
    add_contrib (qpl-cmake qpl) # requires: idxd-config
endif ()
[...]
```

With that, we could omit l. 2-11 in contrib/qpl-cmake/CMakeLists.txt and l. 2-13 in contrib/idxd-config/CMakeLists.txt.

Reply:
Done